frontend/src/components/messstelle/optionsmenue/panels/ZeitPanel.vue

```diff
@@ -64,28 +64,92 @@
                     </p>
                 </v-col>
             </v-row>
+            <v-divider></v-divider>
+            <panel-header
+                font-size="0.875rem"
+                font-weight="bold"
+                padding="10px 0 0 0"
+                header-text="Wochentag"
+            ></panel-header>
+            <tages-typ-radiogroup
+                v-model="chosenOptionsCopy"
+                :is-chosen-tages-typ-valid="isChosenTagesTypValid"
+            />
+            <v-divider></v-divider>
```

Comment:
Macht es Sinn den Panel-Header und Divider mit in die Komponente zu packen?
Betrifft auch alle anderen.